Resources/Prototypes/_Pirate/Entities/Objects/Devices/encryption_keys.yml

```
@@ -0,0 +1,36 @@
+  id: EncryptionKeyInterdyne
+  name: Криптографічний ключ інтердайну
+  description: "Здається цей ключ належить робітнику корпорації Інтердайн"
+  components:
+  - type: EncryptionKey
+    channels:
+    - Interdyne
+    defaultChannel: Interdyne
+  - type: MetaData
+  - type: Sprite
+    layers:
+      - sprite: Objects/Devices/encryption_keys.rsi
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)